docs/src/snoopr.md

```diff
@@ -259,7 +268,7 @@ Variables
   container::Array{AbstractFloat,1}
 
 Body::Int64
-    @ REPL[3]:1 within `callf'
+    @ REPL[3]:1 within callf
```

Comment:
Prints with the backticks for me?

Reply:
This backtick breaks the code blocks of the markdown. I thought removing it does not change the content much 🤔 
Can we escape ` ?